website/static/talks/talkStrings.tsx

```diff
@@ -10,6 +10,17 @@ export type Talk = {
 };
 
 export const Talks: Talk[] = [
+  {
+    conf: "Iceberg Summit 2025",
+    date: "2025-04-09",
+    image: "iceberg_summit_2025.png",
+    title: "Operationalizing Iceberg Metrics with OpenLineage\n",
```

Comment:
Why newline?

Reply:
Started the site locally with and without the newline and can confirm it's not needed, FWIW